README.md

```diff
@@ -19,9 +19,9 @@ documentation].
 
 | Release | Release Date |      EOL      |
 |:-------:|:------------:|:-------------:|
-|   v1.6  | Jan 4, 2022  |   July 2022   |
 |   v1.7  | Mar 22, 2022 |   Sept 2022   |
-|   v1.8  | May 17, 2022 |    Dec 2022   |
+|   v1.8  | May 17, 2022 |    Nov 2022   |
+|   v1.9  | Jul 19, 2022 |    Jan 2023   |
```

Comment:
Should this be 14?
```suggestion
|   v1.9  | Jul 14, 2022 |    Jan 2023   |
```

Reply:
No I'm traveling back from the future.

(Yes, no idea how I got this date.)

Reply:
Probably because it's my birthday 🎉 😂